specification/storagesync/resource-manager/Microsoft.StorageSync/StorageSync/RegisteredServer.tsp

```diff
@@ -21,6 +21,7 @@ model RegisteredServer
     Resource = RegisteredServer,
     KeyName = "serverId",
     SegmentName = "registeredServers",
+    Type = Azure.Core.uuid,
```

Comment:
I think we cannot do this, this might break other SDK.
at least we must sync with other SDK owners.
if you just want the SDK to have uuid type on this name, we should use client tsp.